source/unified-test-format/schema-1.5.json

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
While working on the recent changes, I realised that the original PR that added support for the `observeSensitiveCommands` client entity option didn't create the 1.5 schema version and also didn't add the tests to the unified test format's Makefile.

Reply:
I definitely missed that while reviewing #1015.

@prashantmital: Any idea what happened there? You updated the test file itself to use 1.5 but the PR never included a `schema-1.5.json`. Did you actually make one and just forget to `git add` it?

Reply:
Actually, I didn't create `schema-1.5.json` at all. Completely slipped my mind that this was something I needed to do.